src/pudl/transform/ferc1.py

```diff
@@ -1884,6 +1908,41 @@ def apply_xbrl_calculation_fixes(
                         },
                     },
                 ],
+                "accumulated_provision_for_depreciation_amortization_and_depletion_of_plant_utility": [
+                    {
+                        "calc_component_to_replace": {
+                            "name": "depreciation_amortization_and_depletion_utility_plant_in_service",
+                            # A duplicate of 4 other fields, though this is not explicitly defined in the metadata.
+                            "weight": 1.0,
+                            "source_tables": ["utility_plant_summary_ferc1"],
+                        },
+                        "calc_component_new": {},
+                    },
```

Comment:
`accumulated_provision_for_depreciation_amortization_and_depletion_of_plant_utility` is one of those factoids that shows up in two separate tables with effectively identical calculations. I fully copied all elements of the fixes for this calc including the new `rightsutility` fix from `utility_plant_summary_ferc1`.